app/code/Magento/Catalog/Test/Mftf/Test/AdminProductAttributeLabelDontAllowHtmlTagsTest.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminProductAttributeLabelDontAllowHtmlTagsTest">
+        <annotations>
+            <features value="Catalog"/>
+            <stories value="Product Attribute label must not contain HTML tags"/>
+            <title value="Product Attribute label musts not contain HTML tags"/>
+            <description value="Test whenever HTML tags are allowed for a product attribute label"/>
+            <severity value="CRITICAL"/>
+            <group value="catalog"/>
+        </annotations>
+        <before>
+            <!-- Login as admin -->
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+        </before>
+        <after>
+            <!-- Log out -->
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <!-- Go to Stores > Attributes > Product , click "Add New Attribute" -->
+        <actionGroup ref="AdminOpenProductAttributePageActionGroup" stepKey="openProductAttributePage"/>
+        <click selector="{{AdminProductAttributeGridSection.createNewAttributeBtn}}" stepKey="createNewAttribute"/>
```

Comment:
I suggest replacing these steps by a separate ActionGroup, that will navigate to NewProductAttributePage.
`AdminNavigateToNewProductAttributePageActionGroup`, using this page URL `ProductAttributePage`.